modules/samples/artifacts/0049/README.txt

```diff
@@ -0,0 +1,5 @@
+1. Copy {WSO2SPHome}/samples/artifacts/0008/keyvalue-mapper-sample.siddhi file to {WSO2SPHome}/deployment/siddhi-files
```

Comment:
**{WSO2_SP_Home}/deployment/siddhi-files/** should be updated as **{WSO2_SP_Home}/wso2/worker/deployment/siddhi-files/**